File: data/compilers_comparison/src/examples/first_example/src/func45.c
#include "first_example.h" 
array_t* func45(array_t_param* vars, int loopsFactor) {
   size_t pCounter = vars->size;
   array_t* array23;
   if (pCounter > 0) {
      array23 = vars->data[--pCounter];
      array23->refC++;
      DEBUG_COPY(array23->id);
   } else {
      array23 = (array_t*)malloc(sizeof(array_t));
      array23->size = 403;
      array23->refC = 1;
      array23->id = 23;
      array23->data = (unsigned int*)malloc(array23->size*sizeof(unsigned int));
      memset(array23->data, 0, array23->size*sizeof(unsigned int));
      DEBUG_NEW(array23->id);
   }
   return array23;
}

File: data/multilang_comparison/program_julia/src/func45.jl
include("program_julia_head.jl")
function func45(vars::JArray_param, loopsFactor::Int)::JArray
   pCounter = length(vars.data)
   array23 = nothing
   if pCounter > 1
       array23 = vars.data[pCounter]
       pCounter -= 1
       array23.refC += 1
   else
       array23 = JArray(zeros(Int, 403), 403, 1, 23)
   end
   return array23;
end

